Functions/GithubDownloader.cs

```diff
@@ -1,34 +1,30 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Net;
 
 namespace GTAIVSetupUtilityWPF.Functions
 {
-
-    // hi here, i'm an awful coder, so please clean up for me if it really bothers you
-#pragma warning disable S101 // Types should be named in PascalCase
     public static class GithubDownloader
-#pragma warning restore S101 // Types should be named in PascalCase
     {
-
         private static readonly NLog.Logger Logger = NLog.LogManager.GetCurrentClassLogger();
+        
         public static void Download(string downloadUrl, string destination, string downloadedName)
         {
             try
             {
-                Logger.Debug(" Downloading the selected release...");
-                using (WebClient client = new())
+                Logger.Debug("Downloading the selected release...");
```

Comment:
I put a space before the string intentionally, as otherwise it doesn't look (personally) visually pleasing on the log without any space.

Reply:
Fixed.